src/reynir_correct/settings.py

```diff
@@ -620,13 +555,22 @@ def get_code(wrong_form: str) -> str:
             code = Ritmyndir.get_malsnid(wrong_form)
         if not code:
             code = R_EINKUNN[Ritmyndir.get_eink(wrong_form)]
+        if not code:
+            code = "R001"
         return code
 
+    @staticmethod
+    def get_entry(wrong_form: str, index: int) -> Any:
```

Comment:
Maybe the Any type here could be narrowed, e.g. to `Union[str, int, X, Y...]`

Reply:
...and it is actually possible to be fancy here, doing overloads with Literal[0], Literal[1], etc. and corresponding return types, but that is probably overkill.

Reply:
Done